source/frontend/src/features/projects/reports/models.ts

```diff
@@ -19,3 +22,22 @@ export enum ProjectExportTypes {
   COMPENSATION = 'Compensation Requisition Export',
   AGREEMENT = 'Agreement Export',
 }
+
+const getParameterIdFromOptions = (
```

Comment:
would you mind moving this out to a shared method (since I think your last pr uses this as well). maybe in contactutils?

Reply:
different logic, this one returns an array, the other one returns a string. I thought about it.